lib/utils/keyboard_util.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/cupertino.dart';
+
+class KeyboardUtil {
+  KeyboardUtil._();
```

Comment:
not sure yet how is it in flutter when writing tests, but using static `util` class like this oftentimes caused us trouble in native 🤔

Reply:
Okay, I will take a look at it later in UI tests 👌